recipe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* .
+cp $BUILD_PREFIX/share/gnuconfig/config.* autoconf/helpers/
```

Comment:
These got moved in 2.15 ( https://github.com/netwide-assembler/nasm/commit/0bddd0f1b14aa614e1dd1c14297527dc24dee29c ), which may also be causing issues. So updated this step to copy to the new location

Reply:
+1